Source/GetTime.cpp

```diff
@@ -180,18 +123,18 @@ RakNet::TimeUS GetTimeUS_Windows( void )
 #elif defined(__GNUC__)  || defined(__GCCXML__) || defined(__S3E__)
 RakNet::TimeUS GetTimeUS_Linux( void )
 {
-	timeval tp;
+    RakNet::TimeVal tp;
 	if ( initialized == false)
 	{
-		gettimeofday( &tp, 0 );
+        RakNet::gettimeofday( &tp, 0 );

```

Comment:
Small remark: It would appear raknet uses tabs instead of spaces. We should stick to their guidelines.